backup/pom.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>operator-parent</artifactId>
+        <groupId>com.instaclustr.cassandra-operator</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <properties>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+
+
+        <hk2.version>2.4.0-b31</hk2.version>
+        <jersey.version>2.21</jersey.version>
+        <metrics.version>3.1.0</metrics.version>
+        <guice.version>4.2.0</guice.version>
+        <slf4j.version>1.7.12</slf4j.version>
+        <logback.version>1.1.3</logback.version>
+        <jackson.version>2.8.0</jackson.version>
+        <guava.version>21.0</guava.version>
+        <jetty.version>9.4.7.v20170914</jetty.version>
+        <rabbitmq.version>3.5.4</rabbitmq.version>
+        <thrift.version>0.9.3</thrift.version>
+        <azure.svc-mgmt.version>0.9.3</azure.svc-mgmt.version>
+        <azure.ad.auth.version>1.1.2</azure.ad.auth.version>
+        <azure.rsrc-mgmt.version>1.0.3-SNAPSHOT</azure.rsrc-mgmt.version>
+        <azure.storage.version>5.1.1</azure.storage.version>
+        <shiro.version>1.2.2</shiro.version>
+        <cassandra.driver.version>3.2.0</cassandra.driver.version>
+        <zendesk.client.version>0.5.4</zendesk.client.version>
+        <jira.version>0.5</jira.version>
```

Comment:
A bunch of these probably aren't needed any more.
Additionaly, i think deps like Guice and Guava should be pushed to the parent pom so that every jar has the same version (less stuff on disk).

Reply:
👍 Will update when that change lands